catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -336,4 +341,22 @@ private Metacard getMetacardIfExistsOrNull(String id)
   private interface CheckedSupplier<T> {
     T get() throws Exception;
   }
+
+  private boolean isFormJSONValid(FormTemplate jsonInFormTemplate) {
+    String jsonString = GSON.toJson(jsonInFormTemplate);
+    if (isJSONValid(jsonString)) {
+      return true;
+    }
+    LOGGER.debug("Invalid JSON found: {}", jsonString);
```

Comment:
❓Should we also log a message at `WARN` (but without the JSON string) since it's something a system administrator should know about?